content/en/docs/Development/development.md

```diff
@@ -13,27 +13,41 @@ Here you can find development notes intended for maintainers and guidance for ne
+- [The kairos-agent repository](https://github.com/kairos-io/kairos-agent/) contains the `kairos-agent` code
 - [The packages repository](https://github.com/kairos-io/packages) contains package specifications used by `kairos` while building OS images.
-- [The provider-kairos repository](https://github.com/kairos-io/provider-kairos) contains the kairos provider component which uses the SDK to bring up a Kubernetes cluster with `k3s`. It uses images from `kairos` core to remaster images with `k3s` and the `provider` embedded. This allows to automatically bootstrap Kubernetes cluster. Note, Kairos core in runtime can be extended to add providers or deploy automatically images with the embedded provider.
+- [The provider-kairos repository](https://github.com/kairos-io/provider-kairos) contains the kairos provider component which uses the SDK to bring up a Kubernetes cluster with `k3s`.
 
 ## Build Kairos
 
-To build Kairos you need only Docker installed locally, and there is a convenience script in the root of the repository (`earthly.sh`) which wraps `earthly` inside Docker to avoid to install locally which can be used instead of `earthly` (e.g. `./earthly.sh +iso ...`). However, for daily development, it is strongly suggested to install it in your workstation. The `earthly.sh` script runs `earthly` in a container, and as such there are limitations on image caching between builds.
+To build a Kairos OS you only need Docker. There is a convenience script in the root of the repository `earthly.sh` which wraps `earthly` inside Docker to avoid to install locally which can be used instead of `earthly` (e.g. `./earthly.sh +iso ...`). However, for daily development, it is strongly suggested to install Earthly it on your workstation. The `earthly.sh` script runs `earthly` in a container, and as such there are limitations on image caching between builds.
 
-To build a Kairos ISO, you need to specify the flavor. For example, to build Kairos Alpine with `earthly` installed locally:
+To build a Kairos ISO, you need to specify a few parameters. For example, to build Kairos Fedora with `earthly` installed locally:
 
 ```bash
-earthly -P +iso --FLAVOR=alpine
+earthly +iso \
+  --FAMILY=rhel \
+  --FLAVOR=fedora \
+  --FLAVOR_RELEASE=38 \
+  --BASE_IMAGE=fedora:38 \
+  --MODEL=generic \
+  --VARIANT=core
 ```
 
+- **Flavor**: is the distribution (the name flavor is kept for backwards compatibility). In this case `fedora`
+- **Family**: is a group of distributions that get built in a very similar way. In this case `rhel`, which can build `fedora`, `rockylinux` and `almalinux`.
+- **Flavor release**: is the version of the distribution. In this case `38`
+- **Base image**: is the base image used to build the distribution. In this case `fedora**:38`, but you could use your own image which itself is based on `fedora**:38`.
```

Comment:
what's with the starts in `fedora**:38` ?

Reply:
LOL i went too far with my s/:/:**/g ... good catch thanks